app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/RadioInput/index.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Radio } from '@material-ui/core';
+import { IRadioInputProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+import styled from 'styled-components';
+
+const RadioInput = styled(Radio)`
+  &.MuiRadio-root {
+    padding: 0;
+  }
+`;
+
+export default function({
+  selectedColumns,
+  onSingleSelection,
+  columnDetail,
+  columnIndex,
+}: IRadioInputProps) {
+  return (
+    <RadioInput
+      color="primary"
```

Comment:
this can be moved to app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/InputWidgets/index.tsx
as this is used only in single place
if otherwise plz provide explanation

Reply:
Done